src/test/java/defects/GetDefectsNoData.java

```diff
@@ -26,7 +26,7 @@
     @Steps
     DefectsSteps defectsSteps;
 
-    @Ignore
+    @Ignore ("no data test")
```

Comment:
Is there a bug ref for this being disabled?

Reply:
no, it was disabled long time ago, it does not run on local, maybe we should at some point enable it and see how it goes on CI